src/Converter.ts

```diff
@@ -65,39 +60,37 @@ export default class Converter
      * @param path
      * The path to save the converted file to.
      */
-    public async Start(conversionType: ConversionType, path: string): Promise<void>
-    {
+    public async Start(conversionType: ConversionType, path: string): Promise<void> {
         let htmlCode = await this.Document.Render();
 
-        if (conversionType !== ConversionType.HTML)
-        {
+        if (conversionType !== ConversionType.HTML) {
             let server = (Server.createServer({
-                root: this.DocumentRoot
+                root: this.DocumentRoot,
+                cors: true
             }) as any).server as http.Server;
 
             server.listen(8980, "localhost");
 
-            try
-            {
-                let browser = await Puppeteer.launch();
+            try {
+                let browser = await Puppeteer.launch({ args: ["--disable-web-security"]});
                 let page = await browser.newPage();
                 page.setRequestInterception(true);
-                page.once(
+                page.on(
```

Comment:
Oh... Well I've never been able to debug the website since as long as I was pausing the debugger http-server wasn't responding.

If there is no cleaner way to allow debugging you can feel free to leave it like this.
Could you explain how to debug the hosted website? Would be very cool if I'd be able to do that, too 😄